networking/network_observability/network-observability-operator-release-notes.adoc

```diff
@@ -13,16 +13,65 @@ These release notes track the development of the Network Observability Operator
 
 For an overview of the Network Observability Operator, see xref:../../networking/network_observability/network-observability-overview.adoc#dependency-network-observability[About Network Observability Operator].
 
+[id="network-observability-operator-release-notes-1-3"]
+== Network Observability Operator 1.3.0
+The following advisory is available for the Network Observability Operator 1.3.0:
+
+* link:https://access.redhat.com/errata/RHSA-2023:110622[RHSA-2023:110622 Network Observability Operator 1.3.0]
+
+[id="network-observability-operator-1.3.0-features-enhancements"]
+=== New features and enhancements
+
+[id="multi-tenancy-1.3"]
+==== Multi-tenancy in Network Observability 
+* System administrators can allow and restrict individual user access, or group access, to the flows stored in Loki. For more information, see xref:../../networking/network_observability/installing-operators.adoc#network-observability-multi-tenancy_network_observability[Multi-tenancy in Network Observability].
+
+[id="flow-based-dashboard-1.3"]
+==== Flow-based metrics dashboard 
+* This release adds a new dashboard, which provides an overview of the network flows in your {product-title} cluster. For more information, see xref:../../networking/network_observability/network-observability-overview.adoc#network-observability-dashboards[Network Observability metrics].
+
+[id="must-gather-1.3"]
+==== Troubleshooting with the must-gather tool
+* Information about the Network Observability Operator can now be included in the must-gather data for troubleshooting. For more information, see xref:../../networking/network_observability/troubleshooting-network-observability.adoc#network-observability-must-gather[Network Observability must-gather].
+
+[id="multi-arch-1.3"]
+==== Multiple architectures now supported 
+* Network Observability Operator can now run on an amd64, ppc64le, or arm64 architecture. Previously, it only ran on amd64. 
+
+[id="deprecated-features-1.3"]
+=== Deprecated features
+
+[id="authToken-host"]
+==== Deprecated configuration parameter setting
+The release of Network Observability Operator 1.3 deprecates the `spec.Loki.authToken` `HOST` setting. You must now only use the `FORWARD` setting.
```

Comment:
```suggestion
The release of Network Observability Operator 1.3 deprecates the `spec.Loki.authToken` `HOST` setting. When using the Loki Operator, you must now only use the `FORWARD` setting.
```

(because people might still not use the loki operator, in which case nothing changes for them, they can use DISABLED)

Reply:
Adjusted per your wording suggestion. TY!